<ui version="4.0" >
 <class>FormMatch</class>
 <widget class="QDialog" name="FormMatch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>New match</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>End conditions</string>
     </property>
     <layout class="QFormLayout" name="formLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="checkBox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Score</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="spinBox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="checkBox_2" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Games</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="spinBox_2" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="checkBox_3" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Time limit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="spinBox_3" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="suffix" >
         <string> min</string>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>300</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="2" row="0" column="1" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Maps</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5" >
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" >
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Difficulty:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboDifficulty" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text" >
              <string>Show all</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Easy</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Normal</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Hard</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="treeMaps" >
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="rootIsDecorated" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" >
          <item>
           <widget class="QPushButton" name="btnSelectAll" >
            <property name="text" >
             <string>Select all</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectNone" >
            <property name="text" >
             <string>Select none</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QLabel" name="lblMapName" >
          <property name="font" >
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_3" >
          <property name="fieldGrowthPolicy" >
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Author:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="lblMapAuthor" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Nodes:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLabel" name="lblMapNodes" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Difficulty:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="lblMapDifficulty" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="View" name="view" >
          <property name="minimumSize" >
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="renderHints" >
           <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4" >
          <property name="title" >
           <string>Settings</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Map size:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QSpinBox" name="spinMapScale" >
             <property name="suffix" >
              <string> %</string>
             </property>
             <property name="minimum" >
              <number>25</number>
             </property>
             <property name="maximum" >
              <number>400</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Players</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QTreeView" name="treePlayers" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QPushButton" name="btnAdd" >
          <property name="text" >
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemove" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Start match</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QGraphicsView</extends>
   <header>../../src/view.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnStart</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>treePlayers</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>spinMapScale</tabstop>
  <tabstop>treeMaps</tabstop>
  <tabstop>btnSelectAll</tabstop>
  <tabstop>btnSelectNone</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>118</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>184</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>78</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel" >
     <x>185</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>86</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel" >
     <x>197</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>FormMatch</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>511</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel" >
     <x>257</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
